pkg/expr/sql/frame_db_conv.go

```diff
@@ -40,11 +39,22 @@ func convertToDataFrame(ctx *mysql.Context, iter mysql.RowIter, schema mysql.Sch
 		}
 
 		for i, val := range row {
-			v, err := fieldValFromRowVal(f.Fields[i].Type(), val)
+			// Run val through mysql.Type.Convert to normalize underlying value
+			// of the interface
+			nV, _, err := schema[i].Type.Convert(val)
+			if err != nil {
+				return nil, err
+			}
+
+			// Run the normalized value through fieldValeToRow to normalize
```

Comment:
Typo here? I think this should say:
```suggestion
			// Run the normalized value through fieldValFromRowVal to normalize
```